tests/unit/materializers/test_materializer_registry.py

```diff
@@ -29,7 +29,7 @@ def some_step() -> MyFloatType:
         return MyFloatType(3.0)
 
     with does_not_raise():
-        some_step()()
+        some_step().entrypoint()
```

Comment:
Why `entrypoint` is used here and `call_entrypoint` in the next file? They are all units, so I would expect the same to be used. Does param validation bring any harm here?

Reply:
Does not bring any harm I think, I'll change them all to `call_entrypoint`